### ISSUE TITLE
use threading

### ISSUE THREAD

**Issue by FedericoRollo** [IASRobolab/ias_unitree_camera#3]
It could make sense to parallelize the camera functions to extract different frame (e.g. rgb, depth, ...) this could increase the speed of the publishing. 
I'm not sure that multiple parallel access to the camera could be performed with the UnitreeCamera library.

**Comment by FedericoRollo** [comment]
probably it won't be useful because you cannot access the cameras in parallel so it won't fix anything i guess... let's try the rgb camera along with the depth or the pointcloud and see if that's ok. In such case threading is useless.
